pkg/runtime/runtime.go

```diff
@@ -1329,6 +1329,9 @@ func (a *DaprRuntime) loadComponents(opts *runtimeOpts) error {
 	if err != nil {
 		return err
 	}
+	for _, comp := range comps {
```

Comment:
I think only authorized components should be listed. See lines 1336 --> 1341.

Reply:
@mukundansundar, As per my understanding doing it here will also point out all the components loaded, authorized or not. 
Now if some component is not authorized it will help in debugging that